spiketools/sim/occupancy.py

```diff
@@ -0,0 +1,84 @@
+"""Simulate occupancy."""
+
+import numpy as np
+
+###################################################################################################
+###################################################################################################
+
+def single_trial_occ(num_bins, min_occ, max_occ):
+    """Simulate occupancy for a single trial.
+
+    Parameters
+    ----------
+    num_bins: int
+        Number of spatial bins.
+    min_occ: float
+        Minimum occupancy.
+    max_occ: float 
+        Maximum occupancy.
+
+    Returns
+    -------
+    occupancy: array-like
+        Occupancy for a single trial.
+    """
+
+    return np.random.uniform(min_occ, max_occ, size=num_bins)  
+
+
+def single_uniform_occ(num_bins, scale):
+    """Simulate uniform occupancy for a single trial.
+
+    Parameters
+    ----------
+    num_bins: int
+        Number of spatial bins.
+    scale: float
+        Scale of the occupancy.
+
+    Returns
+    -------
+    occupancy: array-like
+        Occupancy for a single trial.
+    """
+
+    return np.ones(num_bins) * scale
+
+
+def sim_occ_trials(num_trials, num_bins, min_occ, max_occ, use_random=False, uniform_scale=1):
```

Comment:
In other places in the module (e.g. `spiketools.spatial.occupancy`) 'occupancy' is spelled out - so that probably wants to be done in these function names too, for consistency